src/backend/src/prisma/migrations/20230130170407_yarn_prisma_generate/migration.sql

```diff
@@ -0,0 +1,49 @@
+-- CreateEnum
+CREATE TYPE "Task_Priority" AS ENUM ('LOW', 'MEDIUM', 'HIGH');
+
+-- CreateEnum
+CREATE TYPE "Task_Status" AS ENUM ('IN_BACKLOG', 'IN_PROGRESS', 'DONE');
+
+-- CreateTable
+CREATE TABLE "Task" (
+    "taskId" TEXT NOT NULL,
+    "title" TEXT NOT NULL,
+    "notes" TEXT NOT NULL,
+    "deadline" TIMESTAMP(3) NOT NULL,
+    "priority" "Task_Priority" NOT NULL,
+    "status" "Task_Status" NOT NULL,
+    "deletedByUserId" INTEGER,
+    "dateDeleted" TIMESTAMP(3),
+    "createdByUserId" INTEGER NOT NULL,
+    "dateCreated" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "projectId" INTEGER NOT NULL,
```

Comment:
Oh lol there's a second migration. Because we haven't merged it in, I'm a fan of deleting and redoing the migration instead of adding a second one. This might be mildly tricky and require resetting your local database to achieve.